client/src/legacy/LeftAndMain.js

```diff
@@ -1013,8 +1013,9 @@ $.entwine('ss', function($) {
 
     renderModal(isOpen) {
       // Build properties
+      const baseUrl = Config.get('baseUrl').replace(/\/+$/, '');
       const tempid = $('body').data('member-tempid');
-      const url = $.path.addSearchParams(Config.get('baseUrl') + '/CMSSecurity/login', {
```

Comment:
This wasn't quite working because `Config.get('baseUrl')` would be a `/` when your site is hosted in the root of your domain ... which would produce a URL of `//CMSSecurity/login` and be interpretated has `http://CMSSecurity/login` by browsers.